config/dev-realm-spsh.json

```diff
@@ -103,6 +103,7 @@
       }
     ],
     "client": {
+      "nextcloud-bildungsablage-test": [],
```

Comment:
Should this be here?

Reply:
Was asking, because I believe this would be the client-id.
But you use ${KC_NEXTCLOUD_CLIENT_ID} to set the ID for the client, both of them should probably match

Reply:
Oh, yeah, you're right, thanks. Completely overlooked that... I'll fix that and look into your other comments on Monday. 👍